src/mesh/RadioInterface.cpp

```diff
@@ -53,8 +53,10 @@ const RegionInfo regions[] = {
 
     /*
         https://lora-alliance.org/wp-content/uploads/2020/11/lorawan_regional_parameters_v1.0.3reva_0.pdf
+        https://www.arib.or.jp/english/html/overview/doc/5-STD-T108v1_0-E1.pdf
+        https://qiita.com/ammo0613/items/d952154f1195b64dc29f
      */
-    RDEF(JP, 920.8f, 927.8f, 100, 0, 16, true, false, false),
+    RDEF(JP, 920.6f, 927.8f, 10, 0, 13, true, false, false),
```

Comment:
An alternative to this is:
    RDEF(JP, 920.6f, 923.5f, 100, 0, 13, true, false, false),
    
    to use just the frequencies with no airtime restrictions.

Reply:
I think that would be the best option, then you still have 2.9MHz, which is way more than e.g. EU868.

In any case this will be a breaking change though, but if there are not a lot of users in Japan yet, that might not be a big problem.

Reply:
Thanks @GUVWAF , I've updated the patch.